src/index.html

```diff
@@ -13,5 +13,131 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
```

Comment:
`<script>` tag should be placed in the bottom of `<body>`  to avoid the rendering of the rest of html